docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also grant granular permissions for data sources to control who can query and edit them.
+
+Dashboard, folder and data source permissions can be set through the UI or APIs or provisioned through Terraform.
+
+### Role-based access control
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud).
+
+If you find basic organization and server administrator roles are too restrictive, you might want to consider using RBAC features.
+RBAC provides you a fully flexible way of granting, changing, and revoking user read and write access to Grafana resources, such as users, data sources and reports.
+
+RBAC comes with a set of pre-defined roles, such as data source writer, which allows updating, reading, or querying all data sources.
+You can assign these roles to users, teams and service accounts.
+
+Moreover, RBAC allows you to create your own custom roles and edit permissions granted by Grafana's basic roles.
+
+## User synchronization between Grafana and Identity Providers
+
+When connecting Grafana to an Identity Provider, it's important to think beyond just the initial authentication setup. You should also think about synchronizing user bases and roles. Doing so will enable users within a group to share the same configuration, so you won't have to set individual permissions for each user.
+
+### Team sync
+
+Team sync is a feature that allows you to synchronize teams or groups from your authentication provider with teams in Grafana. This means that users who are part of specific teams or groups in LDAP, OAuth, or SAML will be automatically added or removed as members of corresponding teams in Grafana. Whenever a user logs in, Grafana will check for any changes in the teams or groups of the authentication provider and update the user's teams in Grafana accordingly. This makes it easy to manage user permissions across multiple systems.
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+
+> **Note:** Currently, team synchronization occurs only when a user logs in. However, if you are using LDAP, it is possible to enable active background synchronization, which was added to Grafana 6.3. This allows for the continuous synchronization of teams.
+
+### Organization sync
+
+> **Note:** Available in Grafana version 7.0 and later.
+
+Organization sync is the process of binding all the users from an organization in Grafana. This allows to delegate the role of managing users to the IdP. This way, there's no need to manage user access from Grafana since the IdP will be queried eveytime a new user tries to log in.
```

Comment:
```suggestion
Organization sync is the process of binding all the users from an organization in Grafana. This delegates the role of managing users to the identity provider. This way, there's no need to manage user access from Grafana since the identity provider will be queried whenever a new user tries to log in.
```